Debugger/src/Connection/Grpc.php

```diff
@@ -0,0 +1,166 @@
+<?php
+/**
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Debugger\Connection;
+
+use Google\ApiCore\Serializer;
+use Google\Cloud\Core\GrpcRequestWrapper;
+use Google\Cloud\Core\GrpcTrait;
+use Google\Cloud\Debugger\DebuggerClient;
+use Google\Cloud\Debugger\V2\Breakpoint;
+use Google\Cloud\Debugger\V2\Debuggee;
+use Google\Cloud\Debugger\V2\Controller2Client;
+use Google\Cloud\Debugger\V2\Debugger2Client;
+
+/**
+ * Implementation of the
+ * [Google Debugger gRPC API](https://cloud.google.com/debugger/docs/).
+ */
+class Grpc implements ConnectionInterface
+{
+    use GrpcTrait;
+
+    /**
+     * @var Serializer
+     */
+    private $serializer;
+
+    /**
+     * @var Controller2Client
+     */
+    private $controllerClient;
+
+    /**
+     * @var Debugger2Client
+     */
+    private $debuggerClient;
+
+    /**
+     * @param array $config
```

Comment:
Is it possible to add documentation for which values are expected in the `$config` array?

Reply:
I like this idea. For brevity we could do something like the following for gRPC connection classes:

```php 
/**
 * @param array $config [optional] Configuration options. Please see
 *        {@see Google\Cloud\Core\GrpcRequestWrapper::__construct()} for
 *        the available options.
 */
```

And for REST:

```php 
/**
 * @param array $config [optional] Configuration options. Please see
 *        {@see Google\Cloud\Core\RequestWrapper::__construct()} for
 *        the available options.
 */
```

Reply:
Added to both Grpc and Rest connections. When the gax update PR #1045 is merged, I can do a PR to update all the Grpc/Rest connections and incorporate the serializer updates for all the clients.